### DISCUSSION TITLE
General config path

### DISCUSSION THREAD

**Discussion by dflvunoooooo** [joshuar/go-hass-agent#124] [Q&A] [answered]
Is the path for the config files `~/.config/com.github.joshuar.go-hass-agent` or is it `~/.config/go-hass-agent`? The first is created automatically but the second is not. And in the [docs for script sensors](https://github.com/joshuar/go-hass-agent/blob/main/docs/scripts.md) it says `~/.config/go-hass-agent`. So this directory has to be created? Does this also apply for the mqtt part and is #123 related?

**Comment by joshuar** [comment, marked as answer]
Thanks for pointing this out! The documentation is out of date, I'll fix that!  The config path is `~/.config/com.github.joshuar.go-hass-agent` and scripts should go under `~/.config/com.github.joshuar.go-hass-agent/scripts`.

**Reply by dflvunoooooo** [reply]
Perfekt, thank you!
